frontend/src/js/environment/config.local.js

```diff
@@ -0,0 +1,2 @@
+angular.module("evalai-config", [])
+.constant("EnvironmentConfig", {"API":"http://localhost:8000/api/"});
```

Comment:
So, are we not using `config.json` anymore for fetching these names?

Reply:
It would be great if we can fetch these as environment variables instead of hard-coding here. @spyshiv do you know a way so that we can do that?

Reply:
@deshraj Actually we were using [gulp-ng-config](https://github.com/ajwhite/gulp-ng-config/) and that project is also using `gulp-util` so either we have to wait for fixing of [issue](https://github.com/ajwhite/gulp-ng-config/issues/71) of `gulp-util` deprecation or we should go for other alternatives. 
So as a quick fix we can consider this with hardcoded values or we can put `environment` folder in `.gitignore`

Reply:
I think the ideal way would be to put in the `config.json` file that we had earlier. Would be much easier when we want to do the continuous deployment. Can you please move it to the file?

Reply:
Ok. I'll do that